### ISSUE TITLE
Vue 3.1.0 project detected as Vue 2

### ISSUE THREAD

**Issue by Derranion** [vuejs/devtools#682]
On my current project `Vue 3.1.0` is used.

After the recent Vue Dev Tools update it started to detect it as Vue 2 project.

This forced Dev Tools to use some outdated `3.4.45 DevTools` version:
![DevToolPluginVersionDroppedDownSignificantly](https://github.com/user-attachments/assets/bc4e6294-5b76-4e24-827e-09ffa6a3873d)

Current workaround I found is to use legacy v6 plugin:
https://chromewebstore.google.com/detail/vuejs-devtools/iaajmlceplecbljialhhkmedjlpdblhp?utm_source=ext_sidebar?utm_source=ext_sidebar

But would be great to resolve this, not sure why this is happening.

Could be related to options API syntax used in the app but this is still a valid Vue 3 syntax.

Thanks in advance.

**Comment by webfansplz** [comment]
This is a bit strange, do you have multiple instances of vue in your app? This is not related to the options API, which is supported in v7.

**Comment by mrleblanc101** [comment]
The `v3.4.35` in the orange block is not the devtools version, but the installed Vue version 🤔 
You probably specify `^3.1.0` in your package.json, so the installed Vue version is `3.4.5` since it respect the semver of `^3.1.0`

**Comment by alexzhang1030** [comment]
`v3.4.35` is the Vue version instead of vue-devtools version.

If everything works fine that means only vue version detection has some bugs.

**Comment by mrleblanc101** [comment]
Or that they don't know the difference between the minimum requested version in the package.json and the installed version in the lockfile.

**Comment by alexzhang1030** [comment]
`v3.4.35` is your project vue version instead of the current vue-devtools version, if everything works fine that might be a vue  version detection bug.

**Comment by mrleblanc101** [comment]
Or that they don't know the difference between the minimum requested version in the package.json and the installed version in the lockfile.

**Comment by alexzhang1030** [comment]
> Or that they don't know the difference between the minimum requested version in the package.json and the installed version in the lockfile.

You can try devtools functions if they can work properly, looks like problems have been resolved, right?

**Comment by mrleblanc101** [comment]
@alexzhang1030 I'm not OP, I think OP is confused, i don't think there is a bug

**Comment by Derranion** [comment]
> The `v3.4.35` in the orange block is not the devtools version, but the installed Vue version 🤔 Your screenshot clearly shows the UI of v7. You probably specify `^3.1.0` in your package.json, so the installed Vue version is `3.4.5` since it respect the semver limitation.

This is correct, `package.json` version used in the project is `"vue": "^3.1.0"`.

For some reason I thought that Vue Dev Tools version can be found in the plugin info but it is a Vue version instead indeed.

In any case I don't know which exact Vue Dev Tools version is used, plugin itself shows `Version 7.6.3`:
![image](https://github.com/user-attachments/assets/7635892b-ad81-453b-9983-7c03422cc300)

It looks this way:
![devToolsUnknownVersion](https://github.com/user-attachments/assets/d5d17bd5-60b7-44d3-9068-1c347ad72197)

One week ago it looked significantly different.

Not sure if this is expected.

**Comment by mrleblanc101** [comment]
The extension was updated from v6 to v7.
I believe the initial version of v7 is 7.6.0.
7.0.0 to 7.6.0 were in Vue Devtools Next for testing I think.
